osmaxx-symbology/OSMaxx_point_symbols/svg/special_coniferous.svg

```diff
@@ -12,12 +12,12 @@
    width="352.78mm"
    height="352.78mm"
    viewBox="0 0 1250.0079 1250.0079"
-   id="svg23818"
+   id="svg24477"
    version="1.1"
    inkscape:version="0.91 r13725"
-   sodipodi:docname="special_coniferous.svg">
+   sodipodi:docname="special_deciduous.svg">
```

Comment:
I don't care, all `sodipodi` namespaced stuff is inkscape internal anyway, it's not used anywhere.